docs/Containers/Blynk_server.md

```diff
@@ -56,19 +56,19 @@ Everything in ❽:
 * will be replaced if it is not present when the container starts; but
 * will never be overwritten if altered by you.
 
-## How Blynk Server gets built for IOTstack
+## <a name="howBlynkServerIOTstackGetsBuilt"></a>How Blynk Server gets built for IOTstack
 
-### GitHub Updates
+### <a name="dockerHubImages"></a>GitHub Updates 
 
 Periodically, the source code is updated and a new version is released. You can check for the latest version at the [releases page](https://github.com/Peterkn2001/blynk-server/releases/).
  
-### IOTstack menu
+### <a name="iotstackMenu"></a>IOTstack menu
 
 When you select Blynk Server in the IOTstack menu, the *template service definition* is copied into the *Compose* file.
 
 > Under old menu, it is also copied to the *working service definition* and then not really used.
 
-### IOTstack first run
+### <a name="iotstackFirstRun"></a>OTstack first run 
```

Comment:
Introducing a typo. You lost the first character: "OTstack"

Reply:
Can't select, cut, paste and chew gum simultaneously.